assignments/submissions/prime-challenge-lauren.js

```diff
@@ -0,0 +1,27 @@
+// Prime Challenge
+// Given a number between 1-100, return true if the number is a prime.
+
+function isNumberPrime(number) {
+
+  for (var i = 2; i < number; i++) {  // start at 2 because 1 doesn't impact the definition of prime
```

Comment:
Can you think of a potential, simple optimization here? 🤔

Reply:
#mathnerd